main.cpp

```diff
@@ -1,7 +1,7 @@
 #include "main.h"
 #include "rc.h"
 
-double Y(RECT r, double interval, double x)
+double Y(const RECT r, double interval, double x)

```

Comment:
Ovo je i dalje kopija, samo što je `const`. Pročitajte još jednom prethodni komentar.